CONTRIBUTING.md

```diff
@@ -115,6 +115,13 @@ Follow the instructions below on how to install Bundler and setup the database.
 * Start memcached: `memcached`
 * Run the tests: `bundle exec rake`
 
+#### Confirmation emails
+
+* Start rails: `rails s`
+* Sign up for a new user: http://localhost:3000/sign_up
+* Account confirmation email: http://localhost:3000/rails/mailers/mailer/email_confirmation
```

Comment:
I think we need to specify a that the url needs the confirmatrion token.
`ttp://localhost:3000/rails/mailers/mailer/email_confirmation/<confirmation-token>`

Reply:
No. It is url where you can see the preview actual confirmation email. User can just click the confirmation link in the email.

http://guides.rubyonrails.org/action_mailer_basics.html#previewing-emails

Reply:
If i requets that url witout a token I'm getting:

```
ActionView::Template::Error (undefined method `html_safe' for nil:NilClass
Did you mean?  html_safe?):
    3:   <%= t('.welcome_message') %>
    4:   <br />
    5:   <%= link_to t('.confirmation_link'),
    6:     update_email_confirmations_url(@user, token: @user.confirmation_token.html_safe) %>
    7: </p>
```

Reply:
oh i just noticed that error only happens if there's no user pending for signup